storage-manager/src/Cache.h

```diff
@@ -94,6 +101,7 @@ class Cache : public boost::noncopyable
         void reset();
         void validateCacheSize();
         
+        virtual void configListener() override;
     private:
         Cache();
         
```

Comment:
I suggest adding a dtor where the config-listener is deregistered.  Future modders won't think of it.

Reply:
nvm, just noticed it has one and it's doing the right thing.